library.properties

```diff
@@ -1,9 +1,9 @@
-name=JeVe_EasyOTA
-version=2.2.0
-author=Jeroen Vermeulen <info@jeve.eu>
-maintainer=Jeroen Vermeulen <info@jeve.eu>
+name=EasyOTA
+version=3.0.0
+author=Andrius Mikonis <andrius.mikonis@gmail.com>
+maintainer=Andrius Mikonis <andrius.mikonis@gmail.com>
```

Comment:
Same here

Reply:
Adding to authors to library.json was easy, but I don't quite know how to do it with library.property file... maybe add them separated by comma ?
I'll update the PR if it's ok.